gcs/pom.xml

```diff
@@ -138,6 +138,10 @@
       <groupId>com.google.code.gson</groupId>
       <artifactId>gson</artifactId>
     </dependency>
+    <dependency>
```

Comment:
Does not look like we need this dependency here.

Reply:
Right, wasn't sure if the shade includes below needs explicit dependency here.